docs/Design/Classes/Class Declarations.md

```diff
@@ -6,18 +6,19 @@ This improvement adds support for class declarations to GLS, including:
 
 * Naming a class
 * Declaring any generics *(optional)*
-* Declaring a class to extend *(optional)*
+* Declaring a parent to extend/implement *(optional)*
 
 
 ## Commands
 
 ### `class start`
 
-`class start` `:` `classDescriptor`*`[, parentClassDescriptor]`*
+`class start : classDescriptor [, "extends"/"implements" parentDescriptor]`
```

Comment:
`extends` and `implements` are two separate things. IMO the syntax for that should be `class start : classDescriptor[ extends parentClassDescriptor][ implements parentInterfaceDescriptor]`. 

There's no way I can think of to imply the implements and extends can be in either order (and it's a little loose allowing each way), so we can say it must be in that order.